src/Unleash/Unleash.csproj

```diff
@@ -16,31 +16,15 @@
   </PropertyGroup>
 
   <!-- Need to conditionally bring in references for the .NET Framework 4.* targets -->
- <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
```

Comment:
For other reviewers - we decided to drop older versions of .NET here. They're EOL, becoming burdensome to maintain and are blocking for the Yggdrasil wrapper

Reply:
Great! We should be explicit about this and release a major version.
Seeing as we're publishing a [.NET Standard 2.0](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0) theoretically we indirectly still support down to `4.6.2`

Reply:
Correct, however like the doc you shared mentions:

> While NuGet considers .NET Framework 4.6.1 as supporting .NET Standard 1.5 through 2.0, there are several issues with consuming .NET Standard libraries that were built for those versions from .NET Framework 4.6.1 projects. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.

Reply:
@daveleek If I read @nunogois and the linked doc correctly (and I only _think_ I do) then `4.6.2` is out. I'm open to being told we need `4.6.2` but burdensome is what I want to call that and I'm very lazy

Reply:
No I'm fine with them being dropped by us which is what I'm saying in a roundabout way. With Standard 2.0 we're supporting a few runtimes (through using Standard 2.0) even if we drop the explicit support for them, in the cases where that is absolutely necessary. But yes, people on 4.6.1 might experience issues trying to add our library to their projects